pkg/performanceprofile/controller/performanceprofile/components/tuned/tuned_test.go

```diff
@@ -160,34 +162,74 @@ var _ = Describe("Tuned", func() {
 					bootLoader, err := tunedData.GetSection("bootloader")
 					Expect(err).ToNot(HaveOccurred())
 					Expect(bootLoader.Key("cmdline_power_performance").String()).To(Equal(cmdlineHighPowerConsumption))
+					Expect(bootLoader.Key("cmdline_pstate").String()).To(Equal(cmdlineHighPowerConsumptionPstate))
 				})
 			})
 
 			When("realtime workload enabled", func() {
 				It("should not contain idle=poll cmdline", func() {
 					profile.Spec.WorkloadHints = &performancev2.WorkloadHints{
-						RealTime:             pointer.BoolPtr(false),
-						HighPowerConsumption: pointer.BoolPtr(false),
+						HighPowerConsumption: pointer.BoolPtr(true),
+						RealTime:             pointer.BoolPtr(true),
 					}
 					tunedData := getTunedStructuredData(profile)
 					bootLoader, err := tunedData.GetSection("bootloader")
 					Expect(err).ToNot(HaveOccurred())
-					Expect(bootLoader.Key("cmdline_idle_poll").String()).ToNot(Equal(cmdlineIdlePoll))
+					Expect(bootLoader.Key("cmdline_idle_poll").String()).To(Equal(cmdlineIdlePoll))
```

Comment:
do we want to keep the old negative test? we're changing it to positive

Reply:
The test was wrong.

https://github.com/openshift/cluster-node-tuning-operator/blob/59698befa86989be3295f825c2faa3e8c5b5d4bb/pkg/performanceprofile/controller/performanceprofile/components/tuned/tuned_test.go#L169

and the negative one:

https://github.com/openshift/cluster-node-tuning-operator/blob/59698befa86989be3295f825c2faa3e8c5b5d4bb/pkg/performanceprofile/controller/performanceprofile/components/tuned/tuned_test.go#L182

Reply:
This is in general good information to convey in the commit message: why the old test was wrong, and how we are fixing it.